File: Selenium/Activities/Activity7.py
from selenium import webdriver
from selenium.webdriver.common.by import By
with webdriver.Firefox() as driver:
    driver.get("https://training-support.net/webelements/dynamic-controls")
    print(f"the title of the main page is: {driver.title}")

    textField = driver.find_element(By.ID,"textInput")
    textField.is_enabled()
    print(f"the textField is enabled: {textField.is_enabled()}")

    enableButton = driver.find_element(By.XPATH,"//button[text()='Enable Input']")
    enableButton.click()

    print(f"the textFiled is enabled: {textField.is_enabled()}")

    driver.quit()

File: Selenium/Activities/Activity7.java
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity7 {
	public static void main(String args[]) {
		WebDriver driver = new FirefoxDriver();
		driver.get("https://training-support.net/webelements/dynamic-controls");
		System.out.println("The title of the main page is: "+driver.getTitle());
		
		WebElement textField = driver.findElement(By.id("textInput"));
		System.out.println("the textfield is enabled: "+textField.isEnabled());
		

		WebElement enableButton = driver.findElement(By.xpath("//button[text()='Enable Input']"));
		enableButton.click();
		
		System.out.println("the textfield is enabled: "+textField.isEnabled());
		driver.quit();
		
	}

}
